api/src/client/dynamics/fire-safety/DynamicsFireSafetyHealthCheckClient.test.ts

```diff
@@ -0,0 +1,57 @@
+import { DynamicsFireSafetyHealthCheckClient } from "@client/dynamics/fire-safety/DynamicsFireSafetyHealthCheckClient";
+import { AccessTokenClient } from "@client/dynamics/types";
+import { HealthCheckMethod } from "@domain/types";
+import { LogWriter, LogWriterType } from "@libs/logWriter";
+import axios from "axios";
+
+jest.mock("axios");
+jest.mock("winston");
+const mockAxios = axios as jest.Mocked<typeof axios>;
+
+describe("DynamicsFireSafetyHealthCheckClient", () => {
+  let client: HealthCheckMethod;
+  let logger: LogWriterType;
+  let stubAccessTokenClient: jest.Mocked<AccessTokenClient>;
+
+  const ORG_URL = "www.test-org-url.com";
+
+  beforeEach(() => {
+    logger = LogWriter("NavigatorWebService", "ApiLogs", "us-test-1");
+    stubAccessTokenClient = {
+      getAccessToken: jest.fn(),
+    };
+    client = DynamicsFireSafetyHealthCheckClient(logger, {
+      accessTokenClient: stubAccessTokenClient,
+      orgUrl: ORG_URL,
+    });
+  });
+
+  it("returns a passing health check if data can be retrieved sucessfully", async () => {
+    mockAxios.get.mockResolvedValue({});
+    expect(await client()).toEqual({ success: true, data: { message: "OK" } });
+  });
+
+  it("returns a failing health check if unexpected data is retrieved", async () => {
+    mockAxios.get.mockRejectedValue({ response: { status: 404 }, message: "" });
+    expect(await client()).toEqual({
+      success: false,
+      error: {
+        message: "Bad Gateway",
+        serverResponseBody: "",
+        serverResponseCode: 404,
+        timeout: false,
+      },
+    });
+  });
+
+  it("returns a passing health check if axios request times out", async () => {
```

Comment:
[pebble] Is a timeout still considered to be a passing health check? The question applies for other test files as well. Let me know if I'm not understanding this correctly.